indy_node/test/rs_schema/test_rs_send_schema.py

```diff
@@ -0,0 +1,175 @@
+import uuid
+import json
+
+import pytest
+from indy_common.authorize.auth_constraints import AuthConstraintForbidden
+from indy_common.types import SetRsSchemaDataField
+from indy_node.test.api.helper import sdk_write_rs_schema_and_check, build_rs_schema_request
+from plenum.common.exceptions import RequestRejectedException
+
+
+def test_send_rs_schema_multiple_attrib(looper, sdk_pool_handle, sdk_wallet_endorser):
+    _, identifier = sdk_wallet_endorser
+    authors_did, name, version, type = identifier, "ISO18023_Drivers_License", "1.1", "8"
+    _id = identifier + ':' + type + ':' + name + ':' + version
+    schema = {
```

Comment:
I think, that this definition can be moved to fixture of template or something like that for reusing in several places.

Reply:
I will add a `templates.py` for reusing the schema structure.